### ISSUE TITLE
Tap on contacts name should open contacts screen

### ISSUE THREAD

**Issue by trufae** [Antidote-for-Tox/Antidote#281]
Inside the Chat window tapping on the user name at the center-top of the screen does nothing right now.

**Comment by dvor** [comment]
Actually I'm planning to add chat details screen, which will contain some contact information / list of participants (for group chat), some options like "disable notifications for this chat", maybe list of files.

**Comment by trufae** [comment]
Do you have a working group chat support in a branch? i’m really looking forward to see support for it in all the Tox clients. Hopefully before the end of the year :D

**Comment by dvor** [comment]
Nope, I don't have one yet. See my last comment here https://github.com/Antidote-for-Tox/Antidote/issues/200#issuecomment-269078010
